packs/boons-and-curses/abadar-minor-boon.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "deityboon",
         "description": {
-            "value": "<p><strong>Minor Boon (Single Use)</strong> Abadar warns his favored against those who might unfairly take advantage. Once, when someone rolls a success on a Deception check to Lie maliciously to you and you alone, they get a critical failure instead.</p>\n<p>Abadar typically chooses to grant this boon in response to an extremely consequential lie.</p>"
+            "value": "<p>Abadar warns his favored against those who might unfairly take advantage.</p>\n<p>Once, when someone rolls a success on a Deception check to @UUID[Compendium.pf2e.actionspf2e.Item.Lie] maliciously to you and you alone, they get a critical failure instead.</p>\n<p>Abadar typically chooses to grant this boon in response to an extremely consequential lie.</p>"
```

Comment:
Can you make a roll note for this? Lie is a secret action, so the Note should only show up for the GM, but they should be notified that the book could activate.

Reply:
I'm sure we can, since you're the target of the lie rather than the liar.

Reply:
We can ephemeral effect the note